homeassistant/components/sleepiq/__init__.py

```diff
@@ -1,115 +1,87 @@
 """Support for SleepIQ from SleepNumber."""
-from datetime import timedelta
 import logging
 
 from sleepyq import Sleepyq
 import voluptuous as vol
 
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
 from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
 from homeassistant.core import HomeAssistant
-from homeassistant.helpers import discovery
 import homeassistant.helpers.config_validation as cv
-from homeassistant.helpers.entity import Entity
 from homeassistant.helpers.typing import ConfigType
-from homeassistant.util import Throttle
 
 from .const import DOMAIN
-
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=30)
+from .coordinator import SleepIQDataUpdateCoordinator
+from .models import SleepIQHassData
 
 _LOGGER = logging.getLogger(__name__)
 
 CONFIG_SCHEMA = vol.Schema(
     {
-        vol.Required(DOMAIN): vol.Schema(
-            {
-                vol.Required(CONF_USERNAME): cv.string,
-                vol.Required(CONF_PASSWORD): cv.string,
-            }
-        )
+        DOMAIN: {
+            vol.Required(CONF_USERNAME): cv.string,
+            vol.Required(CONF_PASSWORD): cv.string,
+        }
     },
     extra=vol.ALLOW_EXTRA,
 )
 
+DATA_SLEEPIQ = "data_sleepiq"
```

Comment:
Since its shared, please move it to `const.py`

Reply:
Done